plugin.xml

```diff
@@ -55,6 +55,7 @@
     <platform name="ios">
         <!-- required background modes:  App registers for location updates -->
         <hook type="after_plugin_install" src="hooks/add_swift_support.js"/>
```

Comment:
Yes, because you can be adding the plugin once the platform is already installed, in which case this is the hook that will be triggered. Actually I guess this is the most common case.